### ISSUE TITLE
Change of setting auto_lock_old_threads are not saved

### ISSUE THREAD

**Issue by auge8472** [ilosuna/mylittleforum#103]
If I change the value of the setting `auto_lock_old_threads`, it will not be saved to the database. Changes to (i.e.) the settings for `avatar_max_height` and `avatar_max_width` on the other hand are saved.

**Comment by derletztekick** [comment]
This is realted to https://github.com/ilosuna/mylittleforum/issues/98. The _other_ value is saved. It is saved by https://github.com/ilosuna/mylittleforum/commit/c66b6576dc3a02885c3d8f8e2088f46042df6e3d

Done.

**Comment by auge8472** [comment]
I think, I was a bit to fast. I fixed the problem yesterday with #98. **And** I altered the setting for my own forum, but only for one of them. I have one for the theme-development (that I fixed yesterday) and another one to check the new features and bugfixes (that I didn't fix until I reported this issue today).

So it was only a mistake from my side. I corrected the database after reporting this issue and now I can alter the value without problem. I think, I can archive this issue.
